vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,239 @@ lyt3 <- basic_table() %>%
+  analyze(head(names(iris), -1), afun = function(x) {
+    list(
+      "mean / sd" = rcell(c(mean(x), sd(x)), format = "xx.xx (xx.xx)"),
+      "range" = rcell(diff(range(x)), format = "xx.xx")
+    )
+  }, section_div = "+")
+
+build_table(lyt, iris)
+```
+
+Section dividers can be set to " " to create a blank line.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("Species") %>%
+  analyze(head(names(iris), -1), afun = function(x) {
+    list(
+      "mean / sd" = rcell(c(mean(x), sd(x)), format = "xx.xx (xx.xx)"),
+      "range" = rcell(diff(range(x)), format = "xx.xx")
+    )
+  }, section_div = " ")
+
+build_table(lyt, iris)
+```
+
+Separation characters can be specified for different row splits. However, only one will be printed if they "pile up" next to each other. 
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("RACE", section_div = "=") %>%
+  split_rows_by("STRATA1", section_div = "~") %>%
+  analyze("AGE", mean, var_labels = "Age", format = "xx.xx")
+
+build_table(lyt, DM)
+```
+
+### Indent Modifier
+
+Tables by default do not have an indent at the split level. A custom indent value can be supplied with the `indent_mod` argument within a split function.
+
+```{r}
+basic_table(
+  title = "Study XXXXXXXX",
+  subtitles = c("subtitle YYYYYYYYYY", "subtitle2 ZZZZZZZZZ"),
+  main_footer = "Analysis was done using cool methods that are correct",
+  prov_footer = "file: /path/to/stuff/that/lives/there HASH:1ac41b242a"
+) %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX", indent_mod = 3) %>%
+  split_rows_by("STRATA1", indent_mod = 5) %>%
+  analyze("AGE", mean, format = "xx.x") %>%
+  build_table(DM)
+```
+
+### Variable Label Visibility
+
+With split instructions, visibility of the label for the variable being split can be modified to `visible`, `hidden` and `topleft` with the `show_labels` argument, `label_pos` argument, and `child_labels` argument where applicable. Note: this is NOT the name of the levels contained in the variable. For analyze calls, \code{default} indicates that the variable should be visible only if multiple variables are analyzed at the same level of nesting.
+
+Visibility of labels for the groups generated by a split can also be modified using the `child_label` argument with a split call.
+```{r}
+lyt <- basic_table(show_colcounts = TRUE) %>%
+  split_cols_by(var = "ARM") %>%
+  split_rows_by("SEX", split_fun = drop_split_levels, child_labels = "visible") %>%
+  split_rows_by("STRATA1") %>%
+  analyze("AGE", mean, show_labels = "visible")
+
+build_table(lyt, DM) # notice the placement of the "AGE" label.
+
+lyt2 <- basic_table(show_colcounts = TRUE) %>%
+  split_cols_by(var = "ARM") %>%
+  split_rows_by("SEX", split_fun = drop_split_levels, child_labels = "hidden") %>%
+  split_rows_by("STRATA1") %>%
+  analyze("AGE", mean, show_labels = "hidden")
+
+build_table(lyt2, DM) # note that the levels associated with the variable "SEX" no longer appear.
+```
+ 
+```{r}
```

Comment:
Please add text here that calls out what the code is doing and discusses it. Also because we are talking about label_pos here, we should have an example of `label_pos = "topleft"` even though we mentioned it in the top-left section.

Also you are using `page_by` here without discussing what that does at all. Either discuss it or make it not `page_by = TRUE` please.